src/Components/button/button.js

```diff
@@ -0,0 +1,12 @@
+import React from "react";
+import './button.css'
+function Button (props) {
+   
+      return (
+        <button  type="button" onClick={props.fct} className="bg-green-400 hover:bg-green-700 text-white font-bold py-3 px-7 rounded-full " >
```

Comment:
the color doesn't match the one in the figma design try using this one, also add some shadows to match the one in the design, also can you change the `fct` to one make more sense and easily understandable
```suggestion
        <button  type="button" onClick={props.fct} className="bg-primary hover:bg-green-700 text-white font-bold py-3 px-7 rounded-full " >
```